ui-components/menuitems.md

```diff
@@ -127,7 +127,7 @@ A nested menu item is a menu item with a submenu. Registering a submenu this way
 | isDisabled | `() => boolean` | Checks if the menu item is disabled. |
 | setDisabled | `(state: boolean) => void` | Sets the menu item's disabled state. |
 
-#### Example
+#### Example: Creating nested menu items
```

Comment:
Small thing and just an observation. Would this be better as `Creating a nested menu item` to match with other example headings in this file?

Reply:
My bad, misread the example and thought there was more than one nested menu items.
```suggestion
#### Example: Creating a nested menu item
```